app/controllers/likes_controller.rb

```diff
@@ -1,2 +1,25 @@
 class LikesController < ApplicationController
+  before_action :set_user_and_post, only: %i[new create]
+
+  def new
+    @like = Like.new
+  end
+
+  def create
+    @post = @user.posts.find(params[:post_id])
+    @like = @post.likes.new(user: current_user)
+
+    if @like.save
+      redirect_to user_post_path(@user, @post), notice: 'Like was successfully created.'
+    else
+      render :new
+    end
+  end
```

Comment:
- [ ] Nice work creating the **create** action for the likes_controller :clap: However it would be nice to add a button in the post view to allow the user to like a post.